sections/image-banner.liquid

```diff
@@ -33,7 +33,7 @@
   assign full_width = '100vw'
   assign widths = '375, 550, 750, 1100, 1500, 1780, 2000, 3000, 3840'
 
-  if section.settings.image_behavior == 'ambient'
+  if section.settings.image_behavior == 'ambient' or section.settings.image_behavior == 'zoom-in'
```

Comment:
This PR is looking good! I noticed it has these Image Banner changes from https://github.com/Shopify/dawn/pull/2623/ in here too. 

I tested the Image with Text-specific changes, and those all seem to work well in my testing too. 👍